INFO 2025-07-08 19:24:46,914 autoreload 9788 9116 Watching for file changes with StatReloader
INFO 2025-07-08 19:33:56,400 autoreload 18580 11912 Watching for file changes with StatReloader
INFO 2025-07-08 19:34:12,359 basehttp 9788 9828 "GET / HTTP/1.1" 302 0
INFO 2025-07-08 19:34:12,492 basehttp 9788 9828 "GET /dashboard/ HTTP/1.1" 200 16476
INFO 2025-07-08 19:34:20,454 basehttp 9788 9828 "GET /add/ HTTP/1.1" 200 45866
INFO 2025-07-08 19:34:23,776 basehttp 9788 9828 "GET /api/search-food/?q=app HTTP/1.1" 200 423
INFO 2025-07-08 19:34:24,299 basehttp 9788 9828 "GET /api/search-food/?q=apple HTTP/1.1" 200 344
INFO 2025-07-08 19:34:28,361 basehttp 9788 9828 "GET /api/search-food/?q=banan HTTP/1.1" 200 91
INFO 2025-07-08 19:34:28,955 basehttp 9788 9828 "GET /api/search-food/?q=banana HTTP/1.1" 200 91
INFO 2025-07-08 19:34:42,133 basehttp 9788 9828 "POST /add/ HTTP/1.1" 302 0
INFO 2025-07-08 19:34:42,345 basehttp 9788 9828 "GET /dashboard/ HTTP/1.1" 200 17934
INFO 2025-07-08 19:35:05,479 basehttp 9788 9828 "GET /profile/ HTTP/1.1" 200 21106
INFO 2025-07-08 19:35:12,457 basehttp 9788 9828 "POST /profile/ HTTP/1.1" 302 0
INFO 2025-07-08 19:35:12,572 basehttp 9788 9828 "GET /dashboard/ HTTP/1.1" 200 17925
INFO 2025-07-08 19:35:19,575 basehttp 9788 9828 "GET /add/ HTTP/1.1" 200 45866
INFO 2025-07-08 19:35:25,457 basehttp 9788 9828 "GET /api/search-food/?q=bread HTTP/1.1" 200 181
INFO 2025-07-08 19:35:34,890 basehttp 9788 9828 "POST /add/ HTTP/1.1" 302 0
INFO 2025-07-08 19:35:34,979 basehttp 9788 9828 "GET /dashboard/ HTTP/1.1" 200 19212
INFO 2025-07-08 19:35:50,260 basehttp 9788 9828 "GET /profile/ HTTP/1.1" 200 21106
INFO 2025-07-08 19:36:08,353 basehttp 9788 9828 "GET /admin/ HTTP/1.1" 200 8449
INFO 2025-07-08 19:36:08,388 basehttp 9788 9828 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,388 basehttp 9788 10652 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,397 basehttp 9788 19612 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,397 basehttp 9788 14192 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,399 basehttp 9788 8580 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,403 basehttp 9788 16008 "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,403 basehttp 9788 16008 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,424 basehttp 9788 16008 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:36:08,424 basehttp 9788 8580 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:36:22,669 basehttp 9788 21344 "GET / HTTP/1.1" 200 10156
WARNING 2025-07-08 19:36:23,348 log 9788 21344 Not Found: /favicon.ico
WARNING 2025-07-08 19:36:23,348 basehttp 9788 21344 "GET /favicon.ico HTTP/1.1" 404 5373
INFO 2025-07-08 19:36:33,188 basehttp 9788 21344 "GET /register/ HTTP/1.1" 200 8755
INFO 2025-07-08 19:36:42,399 basehttp 9788 21344 "GET /login/ HTTP/1.1" 200 6418
INFO 2025-07-08 19:36:51,171 basehttp 9788 21344 "POST /login/ HTTP/1.1" 302 0
INFO 2025-07-08 19:36:51,263 basehttp 9788 21344 "GET /dashboard/ HTTP/1.1" 200 18823
INFO 2025-07-08 19:37:16,150 basehttp 9788 12316 "GET / HTTP/1.1" 302 0
INFO 2025-07-08 19:37:16,367 basehttp 9788 12316 "GET /dashboard/ HTTP/1.1" 200 18823
WARNING 2025-07-08 19:37:16,880 log 9788 12316 Not Found: /favicon.ico
WARNING 2025-07-08 19:37:16,881 basehttp 9788 12316 "GET /favicon.ico HTTP/1.1" 404 5373
INFO 2025-07-08 19:37:26,414 basehttp 9788 10264 "GET / HTTP/1.1" 200 10156
INFO 2025-07-08 19:37:34,424 basehttp 9788 10264 "GET /register/ HTTP/1.1" 200 8755
ERROR 2025-07-08 19:38:21,754 log 9788 10264 Internal Server Error: /register/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.IntegrityError: UNIQUE constraint failed: calories_userprofile.user_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 29, in register
    UserProfile.objects.create(user=user, daily_calorie_goal=2000)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\query.py", line 663, in create
    obj.save(force_insert=True, using=self.db)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\base.py", line 902, in save
    self.save_base(
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\base.py", line 1008, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\base.py", line 1169, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\base.py", line 1210, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\query.py", line 1868, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1882, in execute_sql
    cursor.execute(sql, params)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 360, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.IntegrityError: UNIQUE constraint failed: calories_userprofile.user_id
ERROR 2025-07-08 19:38:21,770 basehttp 9788 10264 "POST /register/ HTTP/1.1" 500 166748
INFO 2025-07-08 19:38:21,834 basehttp 9788 10264 "POST /register/ HTTP/1.1" 200 9096
INFO 2025-07-08 19:38:28,834 basehttp 9788 10264 "POST /register/ HTTP/1.1" 200 9698
INFO 2025-07-08 19:38:31,091 basehttp 9788 10264 "GET /login/ HTTP/1.1" 200 6418
INFO 2025-07-08 19:38:40,775 basehttp 9788 10264 "POST /login/ HTTP/1.1" 302 0
INFO 2025-07-08 19:38:40,865 basehttp 9788 10264 "GET /dashboard/ HTTP/1.1" 200 16480
INFO 2025-07-08 19:39:08,515 basehttp 9788 10264 "GET /profile/ HTTP/1.1" 200 21112
INFO 2025-07-08 19:39:16,498 basehttp 9788 10264 "POST /profile/ HTTP/1.1" 302 0
INFO 2025-07-08 19:39:16,584 basehttp 9788 10264 "GET /dashboard/ HTTP/1.1" 200 16860
INFO 2025-07-08 19:39:20,965 basehttp 9788 10264 "GET /add/ HTTP/1.1" 200 45868
INFO 2025-07-08 19:39:27,155 basehttp 9788 10264 "GET /api/search-food/?q=appl HTTP/1.1" 200 344
INFO 2025-07-08 19:39:27,592 basehttp 9788 10264 "GET /api/search-food/?q=apple HTTP/1.1" 200 344
INFO 2025-07-08 19:39:39,544 basehttp 9788 10264 "POST /add/ HTTP/1.1" 302 0
INFO 2025-07-08 19:39:39,629 basehttp 9788 10264 "GET /dashboard/ HTTP/1.1" 200 17931
ERROR 2025-07-08 19:39:47,696 log 9788 10264 Internal Server Error: /history/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 168, in calorie_history
    return render(request, 'calories/history.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'dict_lookup'
ERROR 2025-07-08 19:39:47,722 basehttp 9788 10264 "GET /history/ HTTP/1.1" 500 337003
ERROR 2025-07-08 19:39:54,037 log 9788 10264 Internal Server Error: /weekly-report/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 180, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: 'daily_goal|mul:0.8'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 268, in weekly_report
    return render(request, 'calories/weekly_report.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 969, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 894, in __init__
    super().__init__(*args, **kwargs)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 171, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 182, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 897, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'mul'
ERROR 2025-07-08 19:39:54,054 basehttp 9788 10264 "GET /weekly-report/ HTTP/1.1" 500 335850
ERROR 2025-07-08 19:51:18,550 log 9788 5780 Internal Server Error: /history/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 168, in calorie_history
    return render(request, 'calories/history.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'dict_lookup'
ERROR 2025-07-08 19:51:18,558 basehttp 9788 5780 "GET /history/ HTTP/1.1" 500 337003
ERROR 2025-07-08 19:51:55,509 log 9788 5780 Internal Server Error: /weekly-report/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 180, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: 'daily_goal|mul:0.8'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 268, in weekly_report
    return render(request, 'calories/weekly_report.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 969, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 894, in __init__
    super().__init__(*args, **kwargs)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 171, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 182, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 897, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'mul'
ERROR 2025-07-08 19:51:55,526 basehttp 9788 5780 "GET /weekly-report/ HTTP/1.1" 500 335850
INFO 2025-07-08 19:54:37,390 autoreload 7964 9664 Watching for file changes with StatReloader
INFO 2025-07-08 19:54:41,482 basehttp 7964 15092 "GET / HTTP/1.1" 302 0
INFO 2025-07-08 19:54:41,872 basehttp 7964 15092 "GET /dashboard/ HTTP/1.1" 200 18823
WARNING 2025-07-08 19:54:42,530 log 7964 15092 Not Found: /favicon.ico
WARNING 2025-07-08 19:54:42,530 basehttp 7964 15092 "GET /favicon.ico HTTP/1.1" 404 5373
INFO 2025-07-08 19:54:50,935 basehttp 7964 15092 "GET /admin/ HTTP/1.1" 200 8449
INFO 2025-07-08 19:54:50,961 basehttp 7964 15092 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,962 basehttp 7964 19296 "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,965 basehttp 7964 19436 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,969 basehttp 7964 21280 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,970 basehttp 7964 15824 "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,973 basehttp 7964 18000 "GET /static/admin/js/theme.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,977 basehttp 7964 18000 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,991 basehttp 7964 18000 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:54:50,991 basehttp 7964 15824 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,494 basehttp 7964 15824 "GET /admin/calories/food/ HTTP/1.1" 200 69584
INFO 2025-07-08 19:54:55,520 basehttp 7964 15824 "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,521 basehttp 7964 21280 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,524 basehttp 7964 19296 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,537 basehttp 7964 19436 "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,540 basehttp 7964 15092 "GET /static/admin/js/core.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,540 basehttp 7964 19296 "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,540 basehttp 7964 21280 "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,540 basehttp 7964 15824 "GET /static/admin/js/actions.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,543 basehttp 7964 19436 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,570 basehttp 7964 19436 "GET /static/admin/img/search.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,574 basehttp 7964 19436 "GET /static/admin/js/filters.js HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,661 basehttp 7964 18000 "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-07-08 19:54:55,699 basehttp 7964 18000 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,703 basehttp 7964 19436 "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 304 0
INFO 2025-07-08 19:54:55,708 basehttp 7964 19436 "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 304 0
INFO 2025-07-08 21:38:34,369 autoreload 10808 15184 Watching for file changes with StatReloader
INFO 2025-07-08 21:38:44,973 basehttp 10808 4540 "GET / HTTP/1.1" 302 0
INFO 2025-07-08 21:38:45,302 basehttp 10808 14232 "GET /dashboard/ HTTP/1.1" 200 18823
WARNING 2025-07-08 21:38:45,767 log 10808 14232 Not Found: /favicon.ico
WARNING 2025-07-08 21:38:45,767 basehttp 10808 14232 "GET /favicon.ico HTTP/1.1" 404 5373
INFO 2025-07-08 21:38:52,629 basehttp 10808 11484 "GET /profile/ HTTP/1.1" 200 21106
INFO 2025-07-08 21:38:59,479 basehttp 10808 11484 "POST /profile/ HTTP/1.1" 302 0
INFO 2025-07-08 21:38:59,601 basehttp 10808 11484 "GET /dashboard/ HTTP/1.1" 200 19206
ERROR 2025-07-08 21:39:04,574 log 10808 11484 Internal Server Error: /weekly-report/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 180, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: 'daily_goal|mul:0.8'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 268, in weekly_report
    return render(request, 'calories/weekly_report.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 969, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 894, in __init__
    super().__init__(*args, **kwargs)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 171, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 182, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 897, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'mul'
ERROR 2025-07-08 21:39:04,607 basehttp 10808 11484 "GET /weekly-report/ HTTP/1.1" 500 333422
ERROR 2025-07-08 21:39:16,316 log 10808 11484 Internal Server Error: /history/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 168, in calorie_history
    return render(request, 'calories/history.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'dict_lookup'
ERROR 2025-07-08 21:39:16,321 basehttp 10808 11484 "GET /history/ HTTP/1.1" 500 334136
INFO 2025-07-08 21:41:58,781 basehttp 10808 11484 "GET /history/ HTTP/1.1" 200 21684
ERROR 2025-07-08 21:42:04,208 log 10808 11484 Internal Server Error: /weekly-report/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 180, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: 'daily_goal|mul:0.8'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 268, in weekly_report
    return render(request, 'calories/weekly_report.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 969, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 894, in __init__
    super().__init__(*args, **kwargs)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 171, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 182, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 897, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'mul'
ERROR 2025-07-08 21:42:04,230 basehttp 10808 11484 "GET /weekly-report/ HTTP/1.1" 500 333420
INFO 2025-07-08 21:42:09,574 basehttp 10808 11484 "GET /delete/1/ HTTP/1.1" 200 9966
INFO 2025-07-08 21:42:19,001 basehttp 10808 11484 "POST /delete/1/ HTTP/1.1" 302 0
INFO 2025-07-08 21:42:19,170 basehttp 10808 11484 "GET /history/ HTTP/1.1" 200 18623
INFO 2025-07-08 21:42:22,912 basehttp 10808 11484 "GET /delete/2/ HTTP/1.1" 200 9969
INFO 2025-07-08 21:42:26,505 basehttp 10808 11484 "POST /delete/2/ HTTP/1.1" 302 0
INFO 2025-07-08 21:42:26,695 basehttp 10808 11484 "GET /history/ HTTP/1.1" 200 11614
INFO 2025-07-08 21:42:36,533 basehttp 10808 11484 "GET /history/ HTTP/1.1" 200 11236
INFO 2025-07-08 21:42:40,734 basehttp 10808 11484 "GET /profile/ HTTP/1.1" 200 21106
INFO 2025-07-08 21:42:42,501 basehttp 10808 11484 "GET /dashboard/ HTTP/1.1" 200 16476
ERROR 2025-07-08 21:42:46,146 log 10808 11484 Internal Server Error: /weekly-report/
Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 180, in translate_token
    op = OPERATORS[token]
         ~~~~~~~~~^^^^^^^
KeyError: 'daily_goal|mul:0.8'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\calories\views.py", line 268, in weekly_report
    return render(request, 'calories/weekly_report.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 969, in do_if
    condition = TemplateIfParser(parser, bits).parse()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 894, in __init__
    super().__init__(*args, **kwargs)
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 171, in __init__
    mapped_tokens.append(self.translate_token(token))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\smartif.py", line 182, in translate_token
    return self.create_var(token)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\defaulttags.py", line 897, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 705, in __init__
    filter_func = parser.find_filter(filter_name)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\VEDANT\projects\web\Django\calorie_tracker\venv\Lib\site-packages\django\template\base.py", line 615, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'mul'
ERROR 2025-07-08 21:42:46,146 basehttp 10808 11484 "GET /weekly-report/ HTTP/1.1" 500 332327
INFO 2025-07-08 21:45:11,096 basehttp 10808 11484 "GET /weekly-report/ HTTP/1.1" 200 22933
INFO 2025-07-08 21:45:29,705 basehttp 10808 11484 "GET /dashboard/ HTTP/1.1" 200 16476
INFO 2025-07-08 21:45:34,541 basehttp 10808 11484 "GET /history/ HTTP/1.1" 200 11236
INFO 2025-07-08 21:45:38,110 basehttp 10808 11484 "GET /profile/ HTTP/1.1" 200 21106
INFO 2025-07-08 21:46:05,159 basehttp 10808 11484 "GET /weekly-report/ HTTP/1.1" 200 22933
INFO 2025-07-08 21:46:11,686 basehttp 10808 11484 "GET /dashboard/ HTTP/1.1" 200 16476
